.gitignore

```diff
@@ -60,4 +60,5 @@ env[23]/
 
 # Project specific ignores
 static/js/modules/*.js
-static/js/dist/*.js
\ No newline at end of file
+static/js/dist/*.js
```

Comment:
Why not ignore the entire `dist` dir instead of just js files?

Reply:
Good call